packages/manager/modules/logs-to-customer/src/components/subscriptions/SubscriptionEmpty.component.tsx

```diff
@@ -0,0 +1,48 @@
+import { CommonTitle, LinkType, Links } from '@ovh-ux/manager-react-components';
+import {
+  ODS_BUTTON_SIZE,
+  ODS_BUTTON_VARIANT,
+  ODS_TEXT_COLOR_INTENT,
+  ODS_TEXT_LEVEL,
+  ODS_TEXT_SIZE,
+} from '@ovhcloud/ods-components';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { OsdsTile, OsdsText, OsdsButton } from '@ovhcloud/ods-components/react';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+
+const SubscriptionEmpty = () => {
+  const navigate = useNavigate();
+  const { t } = useTranslation('logSubscription');
+
+  return (
+    <OsdsTile rounded inline className="flex flex-col w-full h-fit">
+      <div className="flex flex-col gap-6 pb-4">
```

Comment:
```suggestion
      <div className="flex flex-col gap-6">
```
padding is not needed